src/content/docs/en/recipes/build-forms.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Build HTML Forms in Astro Pages
```

Comment:
Calling out that I changed the title and opening paragraph because "0JS" isn't the most readable, I find. So, instead I put HTML here and added something about not sending JavaScript below. 

Totally open to other suggestions here, but I really do not like how 0JS looks, so I'd really rather avoid using that.

Reply:
This works for me!